kuksa-client/README.md

```diff
@@ -55,36 +55,48 @@ Welcome to Kuksa Client version <some_version>
 
 Default tokens directory: /some/path/kuksa_certificates/jwt
 
-connectj to wss://127.0.0.1:8090
-Websocket connected securely.
+Connecting to VSS server at 127.0.0.1 port 55556 using KUKSA GRPC protocol.
```

Comment:
use the default port 55555, not your own mac port

Reply:
Fixed